roles/backend_setup/tasks/main.yml

```diff
@@ -88,6 +88,29 @@
   tags:
     - fscreate
 
+# When VDO is used, upon creation and activation of thinpools discard is not
+# possible, because of the misaligned discard size between thinpool and VDO.
+# Warning: kernel: device-mapper: thin: Data device (dm-N) max discard sectors
+# smaller than a block: Disabling discard passdown. Is displayed.
+# To overcome this, add command in service file to set
+# /sys/kvdo/max_discard_sectors to 4096 and restart vdo service
+
+- name: Update vdo service to set /sys/kvdo/max_discard_sectors to 4096
+  lineinfile:
+     path: /usr/lib/systemd/system/vdo.service
+     line: 'ExecStartPre=/bin/sh -c "echo 4096 > /sys/kvdo/max_discard_sectors"'
```

Comment:
is this persisted across reboots?

Reply:
> is this persisted across reboots?

This task is editing the file vdo.service, and we are adding the line "echo 4096..." to the service file.
vdo.service will ensure to add 4096 to file /sys/kvdo/max_discard_sectors across reboots.

Reply:
ok, thanks